src/util.rs

```diff
@@ -281,3 +280,7 @@ mod test {
         assert_eq!(compare_versions("0.1.1", "0.2.0"), Some(false));
     }
 }
+
+pub fn get_runtime() -> String {
```

Comment:
Since this is public, it'd be good to have a doc string that mentions that it can be configured by the env var!